test/postgresql.test.js

```diff
@@ -206,6 +206,15 @@ describe('postgresql connector', function () {
           console.warn.restore();
         });
 
+        it('should work', function(done) {
+          Post.find({where: {content: {regexp: '^a/i'}}}, function(err, posts) {
```

Comment:
Hm...good question. `pattern/i` is actually the syntax MySQL uses. I believe I coded defensively, so maybe just don't announce it in the docs, but let it slide?